README.md

```diff
@@ -8,7 +8,16 @@ This repo contains all required resources to collect data from Kubernetes cluste
 
 # Installation
 
-Detailed instructions are available in our [Installation Guide](./deploy/README.md).
+Detailed instructions are available in our Installation Guides below.
+
+Sumo Logic Helm Chart Version
+|:--------
+[0.14.0](./deploy/README.md) 
```

Comment:
we should pin the current version too. Only the latest alpha should link to master branch

Reply:
but thank you for adding this

Reply:
hmm.. I was thinking of doing that as well. Will change it, but then we should be careful when doing find and replace for `v0.xx.x` because that might change this URL as well.